### ISSUE TITLE
The AC needs its own year-round chair

### ISSUE THREAD

**Issue by tobie** [w3c/w3process#692]
There seems to be renewed interest for improving AC engagement. I had started [a thread on this topic](https://lists.w3.org/Archives/Member/w3c-ac-forum/2021OctDec/0284.html) on the AC mailing list late last year, in which I lamented the consequences of tooling and chairing issues on the health of AC's communications. As tooling issues are starting to get addressed in adjacent issues, I'll focus this issue on chairing.

It turns out that [the process mentions a chair for AC meetings](https://www.w3.org/2021/Process-20211102/#ACMeetings) ("generally the CEO"), but not for the rest of the year.

There would be a number of benefits of having a year-round chair for the AC, notably:

- bringing conversations to an actionable close,
- following up on the outcomes,
- liaising with the AB, TAG, and Board,
- keeping the AC up to date on open polls,
- reminding folks to vote,
- enforcing the CEPC,
- etc.

_(I note in passing that @fantasai is doing substantial part of that work pro bono already and thank her for it.)_

Additionally, the [Bylaws](https://www.w3.org/2022/10/w3c-bylaws-20221019.docx) give important roles and responsibilities to the membership. Having a year-round chair would help fulfil those more effectively.

I haven't thought through whether it would make sense for the AB's chair (or someone else in the AB) to act as AC chair, or if the AC should have a dedicated chair (nor about the broader implication of either).

**Comment by michaelchampion** [comment]
Strongly agree. Not sure if this is a Process issue, but since the Team/CEO has failed to act on this suggestion for at least 14 years now (see  https://lists.w3.org/Archives/Member/w3c-ac-forum/2008OctDec/0122.html !) perhaps the Process needs to say more about how the AC actually works.

**Comment by cwilso** [comment]
Again, I'm not sure if this is actionable enough for the Process yet, or if this should go to the AB.

I would think very high on the list of such a "chairing" role would be managing the inclusion of the AC.  I think at this point, we have effectively multiple ACs, as the member meetings run in some Hosts effectively replicate the AC forum, but with boundaries.  We need to figure out how to roll that all back together.

**Comment by michaelchampion** [comment]
> I'm not sure if this is actionable enough for the Process yet, or if this should go to the AB.

Agree.  @tobie I'd suggest moving this issue to https://github.com/w3c/AB-public/. (Is that the right place, Chris?)   And for the AB to make it a (REAL!) priority topic; now that there is a Board to drive the business/financial/organizational matters that have taken up most of the AB's attention for the last few years, the AB should have bandwidth for W3C "cultural" improvement as well as formal Process issues.

**Comment by cwilso** [comment]
Unless it really needs to be public, I'd suggest
https://github.com/w3c/AB-memberonly/.

**Comment by dwsinger** [comment]
Yes, yes. The AC needs 'curation' to make it effective as a body.

**Comment by tobie** [comment]
@michaelchampion [wrote](https://github.com/w3c/w3process/issues/692#issuecomment-1353462472):
> Agree. @tobie I'd suggest moving this issue to https://github.com/w3c/AB-public/.

That works for me. I just don't have the right privileges to move issues around.

**Comment by cwilso** [comment]
I've asked the relevant staff to either move this or give me the privileges to do so.

**Comment by michaelchampion** [comment]
To be clear, I don't strongly care whether it's moved to the public or member-only AB repo, I just think this is more of an AB/AC matter than a Process CG issue at this point.
